client/connect-account-page/payment-methods.tsx

```diff
@@ -28,14 +29,17 @@ const PaymentMethods: React.FC = () => {
 				<img src={ Visa } alt="Visa" />
 				<img src={ MasterCard } alt="MasterCard" />
 				<img src={ AmericanExpress } alt="American Express" />
+				{ wcpaySettings.isWooPayStoreCountryAvailable && (
+					<img src={ WooPay } alt="WooPay" />
+				) }
 				<img src={ ApplePay } alt="Apple Pay" />
 				<img src={ GooglePay } alt="Google Pay" />
-				<img src={ GiroPay } alt="GiroPay" />
 				<img src={ DinersClub } alt="DinersClub" />
```

Comment:
I didn't remove the Diners Club icon because https://github.com/Automattic/woocommerce-payments/pull/3878 asking to replace Cartes Bancaires icons to Diners Club.